api_spec.yaml

```diff
@@ -3246,9 +3246,506 @@ paths:
+                - type: object
+                  required:
+                    - status
+                    - createdNew
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusOK'
+                    createdNew:
+                      type: boolean
+                - type: object
+                  required:
+                  - status
+                  - reason
+                  properties:
+                    status:
+                      type: string
+                      enum: ["CONFIG_VALIDATION_ERROR"]
+                    reason:
+                      type: string
+                - type: object
+                  required:
+                    - status
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusTenantDoesNotExistError'
+        '400':
+          $ref: '#/components/responses/400'
+        
+        '401':
+          $ref: '#/components/responses/401'
+        
+        '402':
+          $ref: '#/components/responses/402'
+
+        '404':
+          $ref: '#/components/responses/404'
+        
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/config/thirdparty/remove:
+    post:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Delete ThirdParty Provider config for a tenant.
+
+        If `tenantId` is specified and is not equal to `defaultTenantId`, SuperTokens subscription license key is required.
+
+        If `tenantId` is not provided, `defaultTenantId` will be used.
+
+      operationId: deleteTenantConfigPOST
+      parameters:
+        - $ref: '#/components/parameters/multitenancyRid'
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - thirdPartyId
+              properties:
+                thirdPartyId:
+                  $ref: '#/components/schemas/thirdPartyId'
+                tenantId:
+                  $ref: '#/components/schemas/tenantId'
+
+      responses:
+        200:
+          description: Remove thirdparty config result
+          content:
+            application/json:
+              schema:
+                oneOf:
+                - type: object
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusOK'
+                    didConfigExist:
+                      type: boolean
+        '400':
+          $ref: '#/components/responses/400'
+
+        '401':
+          $ref: '#/components/responses/401'
+
+        '402':
+          $ref: '#/components/responses/402'
+
+        '404':
+          $ref: '#/components/responses/404'
+
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/config/thirdparty/list:
```

Comment:
add a boolean for if thirdparty is enabled or not as well in this API response -> even if status is != OK